server/v2/cometbft/abci_test.go

```diff
@@ -18,6 +18,8 @@ import (
 	gogotypes "github.com/cosmos/gogoproto/types"
 	"github.com/stretchr/testify/require"
 
+	"reflect"
```

Comment:
Could you run the linter? Weird that it doesn't fail, but the import sorting is wrong